### PR TITLE
Fix visual bug of removal of the wrong team member after deletion

### PR DESCRIPTION
**Pull request by VadimSchmitz**
Work done in this pull request
- Fixed visual bug of removal of the wrong team member after deletion.
- Update teams.md to explain how to add members.

https://www.pivotaltracker.com/story/show/181418526

### CODE REVIEW

File: app/js/components/delete_member.ts [hunk 1]
```diff
@@ -11,8 +11,8 @@ $(() => {
       if (!target.matches('.teams__deleteMemberLink')) return;
       event.preventDefault();
 
-      const td = document.querySelector('.teams__actions') as HTMLTableDataCellElement;
-      td.classList.add('loading');
+      const td = $(target).closest('td');
```
Comment (Badlapje) [inline app/js/components/delete_member.ts:14]: why are you reverting to jQuery here?  We were phasing out that dependency, are we returning to it?

File: app/js/components/delete_member.ts [hunk 2]
```diff
@@ -11,8 +11,8 @@ $(() => {
...
+      td.addClass('loading');
```
Comment (Badlapje) [inline app/js/components/delete_member.ts:15]: Why are you reverting to jQuery here?  We were phasing out it's usage when I was working on it, has that decision been reversed?

Comment (Badlapje) [reply app/js/components/delete_member.ts:15]: You can likely use the target. variable, defined on line 10 to navigate to the closest td and delete it without using jQuery?

File: docs/teams.md [hunk 3]
```diff
@@ -41,11 +41,12 @@ From the cli of your host: `docker-compose exec openconext tail -50  /var/log/te
...
-1. open the openconext docker cli
-2. go to mysql (`mysql` on the cli)
-3. select a uiid for one of the invited people: 
-   1. `use teams;`
-   2. `select invitation_uiid from invitations where id = ;` => fill in the id
-4. accept the invite by going to the following url: `https://teams.vm.openconext.org/invitation/accept/$uiid_of_the_invite` where you replace the last bit with the uiid found in the previous step.
-
-If you use the cli to connect to docker: `docker-compose exec openconext mysql -e 'use teams; select invitation_uiid from invitations where id = ;'` & so on.
+1. Find the invitation_uiid of the user you want to accept `docker-compose exec openconext mysql -e "select * from invitations \G" teams`
+When accepting a new invite a new private window needs to opened, otherwise the following error will show: "You are already a member of this Team."
```
Comment (Badlapje) [inline docs/teams.md:45]: "needs to open" instead of "needs to be opened"

File: docs/teams.md [hunk 4]
```diff
@@ -41,11 +41,12 @@ From the cli of your host: `docker-compose exec openconext tail -50  /var/log/te
...
+2. Accept the invite by going to the following url: `https://teams.vm.openconext.org/invitation/accept/"invitation_uiid"` where you replace the text between the "" with the invitation_uiid found in the previous step.
+
+Once this had been done once you can make the team public so adding new members will be easier.
```
Comment (Badlapje) [inline docs/teams.md:48]: "has" instead of "had"

second "once" can be removed (word 6)

**Review by Badlapje — CHANGES_REQUESTED** [review]
Looks good to me, so the root issue the wrong element was selected for deletion.  

I'd preserve the jquery-less style unless we decided to go back to jQuery (in general for all things Surf we were moving away from jQuery as a dependency).

File: app/js/components/delete_member.ts (hunk 2 above)
Comment (VadimSchmitz) [reply app/js/components/delete_member.ts:15]: I was not informed that jquery was being phased out, I will revert this back to regular js.

File: docs/teams.md (hunk 3 above)
Comment (VadimSchmitz) [reply docs/teams.md:45]: nicely spotted aswell

**Review by Badlapje — APPROVED** [review]
Nice work! :shipit:

**Pull request merged**